package.json

```diff
@@ -22,7 +22,7 @@
     "@fortawesome/free-solid-svg-icons": "6.4.0",
     "@fortawesome/react-native-fontawesome": "0.2.7",
     "@hathor/unleash-client": "0.1.0",
-    "@hathor/wallet-lib": "1.9.0",
+    "@hathor/wallet-lib": "1.10.0",
```

Comment:
Isn't it missing the `package-lock.json` change also?

Reply:
Done 91d3872c22d1367042c0526b5431bd001df5de3d